volume/nfs/nfs.go

```diff
@@ -71,26 +79,68 @@ func (v *Volume) DriverName() string {
 }
 
 func (v *Volume) Path() string {
-	return ""
+	v.m.Lock()
+	defer v.m.Unlock()
+	return v.hostFolder
 }
 
 func (v *Volume) Mount() (string, error) {
-	// The return value from this method will be passed to the container
-	return v.name, nil
-}
+	v.m.Lock()
+	defer v.m.Unlock()
 
-func (v *Volume) Unmount() error {
-	return nil
+	v.usedCount++
```

Comment:
Might want to have a clarifying comment along the lines of "If Mount() fails, Unmount() (and therefore release()) will be called, so we need to increment usedCount before we actually try to mount"